include/alpaka/math/abs/AbsStdLib.hpp

```diff
@@ -29,17 +30,15 @@ namespace alpaka
         {
             //! The standard library abs trait specialization.
             template<typename TArg>
-            struct Abs<
-                AbsStdLib,
-                TArg,
-                std::enable_if_t<std::is_arithmetic<TArg>::value && std::is_signed<TArg>::value>>
```

Comment:
Note: these requirements did not work for `std::complex<T>` either, I think they were just wrong, but didn't make any difference before.

Reply:
Ah, now after CI failed I understand the reason. After I removed that, some compilers issue a warning that `std::abs` call was removed as it is called for unsigned.